contracts/liquidator/Liquidator.sol

```diff
@@ -247,11 +260,17 @@ contract Liquidator is IUniswapV3FlashCallback, PeripheryImmutableState, Periphe
                 recipient: address(this),
                 deadline: block.timestamp,
                 amountIn: swapAmount,
-                amountOutMinimum: amountOutMin,
+                amountOutMinimum: 0,
                 sqrtPriceLimitX96: 0
             })
         );
-        emit Swap(swapToken, baseToken, Exchange.Uniswap, poolFee, swapAmount, amountOut);
+
+        if (amountOut < amountOutMin) {
```

Comment:
Let's leave a note here to explain that we defer the `amountOutMin` check to get better error messaging.

Reply:
added note